tests/unit_tests/test_gameconnection.py

```diff
@@ -545,3 +545,9 @@ async def test_handle_action_invalid(game_connection: GameConnection):
 
     game_connection.abort.assert_not_called()
     game_connection.protocol.send_message.assert_not_called()
+
+async def test_result_format_phantom(game: Game, game_connection: GameConnection):
+    game_connection.connect_to_host = CoroutineMock()
```

Comment:
Is this needed?

Reply:
I'm sorry I don't follow which line you are pointing to.

Reply:
550, The one where you're setting `connect_to_host` to a mock object. I don't think `connect_to_host` is triggered by the `GameResult` command.